ext/pcre/tests/preg_grep_error1.phpt

```diff
@@ -3,20 +3,26 @@ Test preg_grep() function : error conditions - bad regular expressions
 --FILE--
 <?php
 /*
-* Function is implemented in ext/pcre/php_pcre.c
-*/
+ * Function is implemented in ext/pcre/php_pcre.c
+ */
 /*
-* Testing how preg_grep reacts to being passed bad regexes
-*/
+ * Testing how preg_grep reacts to being passed bad regexes
+ */
 echo "*** Testing preg_grep() : error conditions ***\n";
-$values = array('abcdef', //Regex without delimiter
-'/[a-zA-Z]', //Regex without closing delimiter
-'[a-zA-Z]/', //Regex without opening delimiter
-'/[a-zA-Z]/F', array('[a-z]', //Array of Regexes
-'[A-Z]', '[0-9]'), '/[a-zA-Z]/', //Regex string
-);
-$array = array(123, 'abc', 'test');
-foreach($values as $value) {
+$values = [
+    'abcdef', //Regex without delimiter
+    '/[a-zA-Z]', //Regex without closing delimiter
+    '[a-zA-Z]/', //Regex without opening delimiter
+    '/[a-zA-Z]/F',
+    [
+        '[a-z]', //Array of Regexes
+        '[A-Z]',
+        '[0-9]',
+    ],
+    '/[a-zA-Z]/', //Regex string
+];
+$array = [123, 'abc', 'test'];
+foreach ($values as $value) {
     @print "\nArg value is $value\n";
```

Comment:
Why is the print call suppressed here?

Reply:
Otherwise we get `Warning: Array to string conversion in`. What do you think about removing this print, as this is not really necessary?

Reply:
Oh it's because of that nested array... I mean this will break in PHP 9 when we promote this sooooooo, removing it is probably fine

Reply:
Thank you for review. I removed unnecessary strings from those tests such:
- header of a test printed
- `value` print
- `Done` string at the end of test